pkg/gce-pd-csi-driver/controller.go

```diff
@@ -514,6 +515,11 @@ func (gceCS *GCEControllerServer) ValidateVolumeCapabilities(ctx context.Context
 	}, nil
 }
 
+// TODO(#809): Implement logic and advertise related capabilities.
+func (gceCS *GCEControllerServer) ControllerGetVolume(ctx context.Context, req *csi.ControllerGetVolumeRequest) (*csi.ControllerGetVolumeResponse, error) {
+	return nil, errors.New("ControllerGetVolume is not implemented")
```

Comment:
Is there a NOT_IMPL error code we're supposed to return?

Reply:
Ah yes. codes.Unimplemented